scripts/raspberryPiSetup.sh

```diff
@@ -43,17 +43,17 @@ echo "Extracted device major number: ${USB_DEVICE_MAJOR}"
 
 # write these to setupEnvVars.sh
 echo "# Appended from rapsberryPiSetup.sh:" >> setupEnvVars.sh
-echo "DAISYHAT_REPO=${REPO_NAME}" >> setupEnvVars.sh
-echo "DAISYHAT_USER=${USER_NAME}" >> setupEnvVars.sh
-echo "DAISYHAT_GH_TOKEN=${GH_TOKEN}" >> setupEnvVars.sh
-echo "DAISYHAT_DEVICE_MAJOR=${USB_DEVICE_MAJOR}" >> setupEnvVars.sh
+echo "export DAISYHAT_REPO=${REPO_NAME}" >> setupEnvVars.sh
+echo "export DAISYHAT_USER=${USER_NAME}" >> setupEnvVars.sh
+echo "export DAISYHAT_GH_TOKEN=${GH_TOKEN}" >> setupEnvVars.sh
+echo "export DAISYHAT_DEVICE_MAJOR=${USB_DEVICE_MAJOR}" >> setupEnvVars.sh
```

Comment:
Interesting. I had it like this earlier and was facing issues with this. Not sure what the problem was, tbh. I'm not a bash expert but it should work either way. Do you know why it didn't work for you?

Reply:
`export` makes the variable available not just to the current shell, but all processes started from the current shell.  For me the variables were not available to `startDockerContainer.sh`  causing the `docker run` command to fail.

Reply:
weird, it was working for me. Maybe a remainder from some earlier setup attempts. I should wipe my Pi and do the setup again to confirm.
Your explaination sounds very reasonable, thanks for pointing this out